ros2/minimal_integration_test/test/basic_test.cpp

```diff
@@ -0,0 +1,43 @@
+// Description: Test if a simple task plan works
+
+#include <rclcpp/rclcpp.hpp>
+
+#include <gtest/gtest.h>
+#include <stdlib.h>
+
+namespace minimal_integration_test {
+class TaskPlanningFixture : public testing::Test {
+ public:
+  TaskPlanningFixture()
+      : node_(std::make_shared<rclcpp::Node>("basic_test"))
+  {
+    RCLCPP_ERROR_STREAM(node_->get_logger(), "DONE WITH CONSTRUCTOR!!");
+  }
+
+  void SetUp() override {
+    // Setup things that should occur before every test instance should go here
+    RCLCPP_ERROR_STREAM(node_->get_logger(), "DONE WITH SETUP!!");
+  }
+
+  void TearDown() override {
+    std::cout << "DONE WITH TEARDOWN" << std::endl;
```

Comment:
Feels like we should use roslog **or** `std::cout` rather than mix them for examples